htdocs/api/v0.0.3-dev/candidates/visits/dicoms/Dicom.php

```diff
@@ -0,0 +1,149 @@
+<?php
+/**
+ * Handles API requests for the candidate's visit dicom tar file
+ *
+ * PHP Version 5.5+
+ *
+ * @category Main
+ * @package  API
+ * @author   Mouna Safi-Harab <mouna.safiharab@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+namespace Loris\API\Candidates\Candidate\Visit\Dicoms;
+require_once __DIR__ . '/../../Visit.php';
+/**
+ * Handles API requests for the candidate's visit dicom tar file. Extends
+ * Visit so that the constructor will validate the candidate
+ * and visit_label portion of URL automatically.
+ *
+ * @category Main
+ * @package  API
+ * @author   Mouna Safi-Harab <mouna.safiharab@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class Dicom extends \Loris\API\Candidates\Candidate\Visit
+{
+    /**
+     * Construct a Dicom class object to output the candidate visits dicom tar file
+     *
+     * @param string $method     The method of the HTTP request
+     * @param string $CandID     The CandID to be serialized
+     * @param string $VisitLabel The visit label to be serialized
+     * @param string $Tarname    The Tar name to be retrieved
+     */
+    public function __construct($method, $CandID, $VisitLabel, $Tarname)
+    {
+        ob_start();
```

Comment:
why is this here? I'm not sure what it's trying to do, but it seems a little convoluted.

Reply:
was there in images/image.php, and the dicom here is clearly a copy of the image one! I tested the download after removing this line (as well as Line 73 below); seems to be fine, so will delete those lines.